src/views/List.jsx

```diff
@@ -12,35 +13,50 @@ export function List({ data }) {
 		item.name.toLowerCase().includes(searchItem.toLowerCase()),
 	);
 
+	const listName = listPath.slice(29);
```

Comment:
We should avoid using magic numbers here! Perhaps a better approach would be to split the path using a delimiter and assign the name to what comes after the delimiter!